.github/workflows/ci.yml

```diff
@@ -36,8 +36,8 @@ jobs:
       env:
         GO111MODULE: off
       run: |
-        go get -u github.com/vbatts/git-validation
-        go get -u github.com/kunalkushwaha/ltag
+        go get -u -v github.com/vbatts/git-validation
+        go get -u -v github.com/kunalkushwaha/ltag
```

Comment:
Can we use `go install`?

Reply:
+1. Was kinda confused how this even worked as it's explicitly blocked starting in 1.18, but then noticed we set go111module=off

Reply:
Thanks; updated!